CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### Fixes
 
+- **General**: Allow to remove the finalizer even if the ScaledObject isn't valid ([#4396](https://github.com/kedacore/keda/issue/4396))
```

Comment:
Is this general or related to the validation?

Reply:
> Is this general or related to the validation?

I don't get your question. Currently, the ScaledObject can't be removed if the validation fails, so I'm not sure. The error is that validation should allow to remove the finalizers, but the ScaledObject is locked at cluster level for everything

Reply:
So the change is specific to validation =)

In that case:
```suggestion
- **Admission Webhooks**: Allow to remove the finalizer even if the ScaledObject isn't valid ([#4396](https://github.com/kedacore/keda/issue/4396))
```